app/controllers/pages_controller.rb

```diff
@@ -14,4 +14,8 @@ def set_headers
   def root_path?
     request.path == root_path
   end
+
+  def home
```

Comment:
This is a generic helper that should not belong to any specific controller action. I would move the lines 22-30 in `app/controllers/hiring_staff/base_controller.rb` together with
```
helper_method :current_user
helper_method :current_school
```
into `app/controllers/application_controller.rb`